jitbuilder/release/src/TransactionalOperations.cpp

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
```

Comment:
The test will hit the success pathways ie the TSTART and TCOMMIT - I am not sure it tests anything related to TABORT. Arranging a test of TABORT will be much harder, but perhaps that should be considered to ensure full functional validation?